samples/simple.js

```diff
@@ -4,12 +4,12 @@ const path = require('path');
 
 const nconf = require('nconf');
 
-require('./../lib/nconf_dir');
+require('./../lib/nconfetti');
 
 nconf.argv()
   .env('__');
 
-nconf.use('Nconfdir', {dir: path.join(__dirname, '..', 'tests', 'configs', 'env'), env: 'development'});
+nconf.use('Nconfetti', {path: path.join(__dirname, '..', 'tests', 'configs', 'env'), env: 'development'});
```

Comment:
wouldn't this be easier to read with `path.resolve(__dirname, '../tests/configs/env');`?